test/Microsoft.SqlTools.ServiceLayer.IntegrationTests/DacFx/DacFxServiceTests.cs

```diff
@@ -842,24 +848,35 @@ private bool ValidateStreamingJobErrors(ValidateStreamingJobResult expected, Val
         private bool ValidateOptions(DeploymentOptions expected, DeploymentOptions actual)
         {
             System.Reflection.PropertyInfo[] deploymentOptionsProperties = expected.GetType().GetProperties();
+            var optionsMapTable = new Dictionary<string, DeploymentOptionProperty<bool>>();
```

Comment:
does this need to get initialized here if it gets set to something else on line 874?

Reply:
yes, since the validation is taking care outside the for loop and optionsMapTable is collecting from the properties foreach loop, needed an initialization . so that I can do another iterate on optionsMapTable for the validation purpose. Thanks.

Reply:
Noticed the validation is still inside, will move outside the loop for better complexity!